common-template/src/main/resources/templates/slack/Errata/beta/newSubscriptionEnhancementErrata.json

```
@@ -0,0 +1,42 @@
+{
+    "blocks": [
+        {
+            "type": "section",
+            "text": {
+                "type": "mrkdwn",
+                "text": "*Enhancements - Errata - Subscription Services*"
+            }
+        },
+        {
+            "type": "section",
+            "text": {
+                "type": "mrkdwn",
+                "text": "There are {data.events.size()} {#if data.events.size() == 1}enhancement{#else}enhancements{/if} affecting your subscriptions.{#if data.events.size() > 3}\nFirst 3 are:{/if}"
```

Nitpick which is valid for most beta templates:
```suggestion
                "text": "There are {data.events.size()} enhancement{#if data.events.size() > 1}s{/if} affecting your subscriptions.{#if data.events.size() > 3}\nFirst 3 are:{/if}"
```
Feel free to make that change or not!